rust/kernel/kernelh.rs

```diff
@@ -19,8 +15,13 @@ pub fn _kasprintf_1(gfp: bindings::gfp_t, fmt: *const c_char, arg1: *const c_cha
 
 /// The `printk` function is a wrapper around the `printk` function from the kernel.
 #[inline]
-pub fn _kasprintf_2(gfp: bindings::gfp_t, fmt: *const c_char, arg1: *const c_char,arg2: *const c_char) -> *mut c_char {
-    unsafe { bindings::kasprintf(gfp, fmt, arg1,arg2) }
+pub fn _kasprintf_2(
+    gfp: bindings::gfp_t,
+    fmt: *const c_char,
+    arg1: *const c_char,
+    arg2: *const c_char,
+) -> *mut c_char {
+    unsafe { bindings::kasprintf(gfp, fmt, arg1, arg2) }
```

Comment:
Add a comment for the unsafe usage.

Reply:
Thanks for reviewing! Sorry, I don't know what this piece of code does; I merely executed a formatting command to change its layout.

Reply:
The safety comment should illustrate why this function is safe to use here. IMO, it should be "FFI call". You can have a look about other safety comments.

Reply:
I didn't write these codes, I just ran `make LLVM=1 rustfmt` to formate these codes. So I'd better not comment on these codes.